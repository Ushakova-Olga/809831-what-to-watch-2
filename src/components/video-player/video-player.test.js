import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from '../video-player/video-player';

it(`Video player correctly renders`, () => {
  const tree = renderer
    .create(<VideoPlayer
      src={``}
      img={``}
      isPlaying={false}
      clickHandler={()=>{}}
      key={``}>
    </VideoPlayer>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
