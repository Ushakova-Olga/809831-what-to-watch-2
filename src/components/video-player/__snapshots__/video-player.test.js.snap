// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video player correctly renders 1`] = `
<video
  height="100%"
  muted={true}
  poster=""
  width="100%"
>
  <source
    src=""
    type="video/null"
  />
</video>
`;
